charts/pelorus/templates/dashboard-snapshot-cronjob.yaml

```diff
@@ -38,7 +38,14 @@ subjects:
   namespace: {{ .Release.Namespace }}
 
 ---
+{{- /*
+Issue #386 - we need to use different APIs
+*/}}
+{{- if .Capabilities.APIVersions.Has "batch/v1/CronJob" -}}
 apiVersion: batch/v1
+{{- else if .Capabilities.APIVersions.Has "batch/v1beta1/CronJob" }}
+apiVersion: batch/v1beta1
+{{- end }}
```

Comment:
Looks like this accidentally got included, but otherwise LGTM.

Reply:
Merged the PR that had this, so if you rebase this should disappear.

Reply:
Yes I created this branch on top of previous rather then master.